core/src/epicli/data/common/defaults/configuration/kafka.yml

```diff
@@ -73,6 +73,12 @@ specification:
     conf_dir: /opt/kafka/config
     data_dir: /var/lib/kafka
     log_dir: /var/log/kafka
+    socket_settings:
+      network_threads: 3 # The number of threads handling network requests
+      io_threads: 8 # The number of threads doing disk I/O
+      send_buffer_bytes: 102400 # The send buffer (SO_SNDBUF) used by the socket server
+      receive_buffer_bytes: 102400 # The receive buffer (SO_RCVBUF) used by the socket server      
+      request_max_bytes: 104857600 # The maximum size of a request that the socket server will accept (protection against OOM)
```

Comment:
Shouldn't those parameters be placed in file roles/kafka/defaults/main.yml?

Reply:
Agree with @plirglo if it shouldn't be changed by user.

Reply:
User should be able to change those parameters. There are users that are running Kafka and having bigger messages than allowed by default - that is why it was added.